python-sdk/src/astro/databases/aws/redshift.py

```diff
@@ -238,12 +238,12 @@ def merge_table(
 
         source_to_target_map_source_columns = list(source_to_target_columns_map.keys())
         source_to_target_map_target_columns = list(source_to_target_columns_map.values())
-        source_table_all_columns = self.hook.run(
+        source_table_all_columns = self.hook.run(  # skipcq BAN-B608
```

Comment:
This skip should we ok since we are wrapping table name with `''`

Reply:
But why are we not just using `params` instead? Or does `self.hook.run` not have this option?

Reply:
good point, not sure how the redshift connector handles it but let me try to fix it

Reply:
Look like the way the redshift connector accepts the param is depend on the value of `paramstyle` 
 https://github.com/aws/amazon-redshift-python-driver/blob/master/redshift_connector/cursor.py#L201 the default value is `format` so even if I'll do `"insert {}".format(table)` it would be string looks difficult to pass param in hook run method for redshift

Reply:
But [the tests passed](https://github.com/astronomer/astro-sdk/actions/runs/3417479740). So why the revert? It is recommended to use `parameters` to handle them properly. By doing that we let the underlying library decide how to deal with it and ensure security.

Reply:
It didn't the test was failing here https://github.com/astronomer/astro-sdk/actions/runs/3417479740/jobs/5689302000   so reverted

Reply:
My bad parameters is only for passing arguments - not table names to sql.